bitbots_quintic_walk/include/bitbots_quintic_walk/walk_node.h

```diff
@@ -101,7 +101,7 @@ class WalkNode {
   /**
    * This method computes the next motor goals and publishes them.
    */
-  void calculateAndPublishJointGoals(const WalkResponse &response);
+  void calculateAndPublishJointGoals(const WalkResponse &response, double dt);
 
   double getTimeDelta();
```

Comment:
please use trailing return type for this function means auto instead of double.

Reply:
we don't want to use auto